### PR TITLE
Lab5 ipynb added

### PR DESCRIPTION
**Pull request by rishab231**
Lab5 ipynb has been created

The images folder should be in the same folder as the notebook. There are no subfolders within the images folder i.e. images folder contains all the images.

### CODE REVIEW

**Comment by akmorrow13** [comment]
@rishab231 please remove duplicate image files. They are already available in the image folder.

File: exercises/lab5/lab5.ipynb [hunk 1]
```diff
@@ -2,232 +2,319 @@
...
+    "deletable": true,
+    "editable": true
+   },
    "source": [
     "For part 1 of this lab, we will research different HLA-A allele frequencies in a specific population of interest.\n",
     "1. Navigate to [Allele Frequencies](http://www.iedb.org/).\n",
     "2. We will use this portal to explore HLA-A frequencies. In the sidebar menu, hover above **HLA** and click on **HLA Allele Freq (Classical)**.\n",
     "2. In the resulting search page, under locus, select **locus A**.\n",
-    "<img src=\"images/locus.png\" width=\"100\" />\n",
+    "<img src=\"../images/lab5/locus.png\" width=\"100\" />\n",
```
Comment (akmorrow13) [inline exercises/lab5/lab5.ipynb]: I still dont think this is going to work. Have you tested in datahub?

Comment (rishab231) [reply exercises/lab5/lab5.ipynb]: Yes I have tested it and it works

File: exercises/lab5.md [hunk 2]
```diff
@@ -1,195 +0,0 @@
-# Lab 5: How the Immune System Detects Mutations
```
Comment (akmorrow13) [inline exercises/lab5.md:1]: Can you not delete this file?

**Review by akmorrow13 — CHANGES_REQUESTED** [review]

File: exercises/lab5/lab5.ipynb (hunk 1 above)
Comment (akmorrow13) [reply exercises/lab5/lab5.ipynb]: how did you upload it to datahub?

**Pull request merged**